### PR TITLE
graph homework

### PR DESCRIPTION
**Pull request by SheyReiff**
Please find attached the screenshot of the graph. My oval is not right on top of my arch, I was wondering if that is how it's supposed to be, or not? 

Thank you! 
Screenshot 2024-02-25 at 11 33 54 AM

### CODE REVIEW

File: src/main/java/reiff/projectile/Projectile.java [hunk 1]
```diff
@@ -39,5 +45,9 @@ public double getInterceptX() {
         double timeToGround = (2 * velocity * Math.sin(radians)) / 9.8;
         return Math.cos(radians) * velocity * timeToGround;
     }
+
+    public double getTotalTime() {
+        return (2 * velocity * Math.sin(radians)) / 9.8;
```
Comment (andrewoid) [inline src/main/java/reiff/projectile/Projectile.java]: getApexTime() * 2

File: src/main/java/reiff/projectile/ProjectileGraph.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        g.translate(0, getHeight());
+        g.setColor(Color.BLACK);
+
+        double interval = 0.1;
+
+        double time = 0;
+        double prevX = 0;
+        double prevY = 0;
+
+        while (time <= projectile.getTotalTime()) {
```
Comment (andrewoid) [inline src/main/java/reiff/projectile/ProjectileGraph.java]: Use a for loop

File: src/main/java/reiff/projectile/ProjectileGraph.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            prevX = x;
+            prevY = y;
+
+            time += interval;
+        }
+
+        g.setColor(Color.BLUE);
+        double x = projectile.getX(projectile.getTotalTime() / 2);
+        double y = projectile.getPeakY();
+        g.fillOval((int) x, -(int) y, 10, 10);
```
Comment (andrewoid) [inline src/main/java/reiff/projectile/ProjectileGraph.java]: The oval doesn't get drawn with the center at x,-y.

The oval gets drawn with the upper left hand corner being at x,y and the width and height being 10.

If you want the center at x,y you need to modify the point.

File: src/main/java/reiff/projectile/Projectile.java (hunk 1 above)
Comment (SheyReiff) [reply src/main/java/reiff/projectile/Projectile.java]: What is this referring to?

Comment (andrewoid) [reply src/main/java/reiff/projectile/Projectile.java]: Take a look at the code for this method and the code for getApexTime

**Comment by andrewoid** [comment]
Replace the screenshot

**Comment by SheyReiff** [comment]
Screenshot 2024-02-25 at 1 13 29 PM

**Review by andrewoid — APPROVED** [review]

**Pull request merged**